cmd/cli/main.go

```diff
@@ -105,6 +105,13 @@ func main() { //nolint
 					Error(err).
 					Msg("Fail to update from version 2.9")
 			}
+		case "2.11":
```

Comment:
You should also add 2.11 to `moiraValidVersions`, otherwise the `checkValidVersion` will fail

Reply:
https://github.com/moira-alert/moira/pull/1022/commits/fa50c3d1a2ee5d34af95ed666c9219de70e0b0db